app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -206,3 +211,81 @@ export const downloadLogAction = task => dispatch => {
     })
   });
 };
+
+// *****************************************************************************
+// * CANCEL MIGRATION TASKS
+// *****************************************************************************
+export const cancelPlanRequestTasksAction = (url, tasks) => dispatch => {
+  const completedTasks = [];
+  const incompleteTasks = [];
+  tasks.map(t => (t.completed ? completedTasks.push(t) : incompleteTasks.push(t)));
```

Comment:
Since you don't use `map`'s return value here, it could be a `forEach` instead.. but it really doesn't matter, and if this is the only thing I can find to comment on, I think your code looks great!